perf-testing/stats.rb

```diff
@@ -1,9 +1,16 @@
 module Stats
-  def p50(data)
-    data[(data.length - 1) / 2]
+  MEASUREMENTS = ['Total request time (ms)', 'Serialization time (ms)', 'Deserialization time (ms)',
+                  'Request payload size (bytes)', 'Response payload size (bytes)'].freeze
+
+  def get_measurements
```

Comment:
Ruby style - its generally considered reasonable to access constants directly rather than defining a method that returns them.  Also ruby method naming usually prefers shorter names and drops the `get`.